src/components/recordset.tsx

```diff
@@ -234,6 +236,27 @@ const RecordsetInner = ({
     });
   };
 
+  /**
+   * On window focus, remove request and update the page
+   */
+  const updateRecords = () => {
```

Comment:
Let's call it `onWindowFocus` or simply `onFocus` so it's easier to understand how this function is used.